types/wlr_output_layout.c

```diff
@@ -52,6 +69,7 @@ static void output_layout_output_destroy(
 	wl_list_remove(&l_output->state->commit.link);
 	wl_list_remove(&l_output->state->output_destroy.link);
```

Comment:
Now that `l_output` is an addon, maybe we can get rid of the `output_destroy` listener?